misc-scripts/xref_mapping/XrefParser/RefSeqCoordinateParser.pm

```diff
@@ -23,405 +23,300 @@ use strict;
 use warnings;
 use Carp;
 use DBI;
-
-use base qw( XrefParser::BaseParser );
+use Readonly;
 use Bio::EnsEMBL::Registry;
 
-sub run_script {
+use parent qw( XrefParser::BaseParser );
+
+# set the score thresholds
```

Comment:
Can we get something a bit more descriptive here? What is the penalty for? Why is there a threshold?